src/app/public/modules/vertical-tabset/vertical-tab.component.html

```diff
@@ -2,7 +2,7 @@
   class="sky-vertical-tab"
   [attr.aria-controls]="ariaControls"
   [attr.aria-invalid]="ariaInvalid"
-  [attr.aria-required]="ariaRequired"
+  [attr.aria-required]="ariaRequired ? ariaRequired : null"
```

Comment:
The `setTimeout` for this bug actually showed an a11y issue here. We default the `role` property to `tab`; however, `aria-required` is not allowed for that role and so our a11y test started failing with the new different timing here. This change just ensures that we aren't adding `aria-required` to the DOM when it is `false`. WIthout this change, we were adding the property to the DOM even when it wasn't needed and that could cause a11y issues with an attribute being added that isn't allowed via `WAI-ARIA` specs.

Reply:
Thanks for the explanation@